modules/apps/transfer/keeper/relay_test.go

```diff
@@ -149,16 +157,92 @@ func (suite *KeeperTestSuite) TestSendTransfer() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestSendTransferDoesNotSetTotalEscrowAmountForSourceIBCToken() {
```

Comment:
This function name got me thinking for a bit can we change it to something like `TestSendTransferNativeTotalEscrowAmount` or anything shorter?

Reply:
I shortened it a bit now to `TestSendTransferSetsTotalEscrowForSourceIBCToken` :)